### PR TITLE
Post Release 5.0.0-beta3 Cleanup

### PR DESCRIPTION
**Pull request by nickclark2016**
Reverts the version back to 5.0.0-dev in the build, updates copyright year in `src/host/premake.h`

### CODE REVIEW

**Review by Jarod42 — APPROVED** [review]

File: src/host/premake.h
Comment (Jarod42) [inline src/host/premake.h:15]: How about having version.h.in using `git describe`?

Something like:
```
local function template(file_in, dict, file_out)
	local content = io.readfile(file_in)
	for k,v in pairs(dict) do
		content = content:gsub(k, v)
	end
	os.mkdir(path.getdirectory(file_out))
	local f, err = os.writefile_ifnotequal(content, file_out)

	if (f == 0) then
		-- file not modified
	elseif (f < 0) then
		error(err, 0)
		return false
	elseif (f > 0) then
		print("Generated " .. file_out)
	end
	return true
end

local function autoversion_h()
	local git_tag, errorCode = os.outputof("git describe --tag --always")
	if errorCode ~= 0 then
        error("`git describe --tag` failed with error code", errorCode, git_tag)
    end
    print("git description: ", git_tag)
	return template(
		"src/autoversion.h.in",
		{["@GIT_DESC@"] = git_tag},
		path.join("solution", _ACTION, "autoversion.h")
end

gitintegration "Always" -- to add git hook

if _ACTION ~= nil then -- Do nothing for premake --version :-)
    autoversion_h()
end
```
So no longer need to modify the code, it just use git tag.

Comment (nickclark2016) [reply src/host/premake.h:15]: Let's table this for a follow up, just so we can complete the current release cycle under the existing release guide. Then we can discuss updating this for the future in maybe a GH discussion?

Comment (Jarod42) [reply src/host/premake.h:15]: No pb to postpone the discussion (I approved BTW).

Comment (nickclark2016) [reply src/host/premake.h:15]: Unfortunately, you're not a reviewer with write access so I have to wait for another review :(

**Review by starkos — APPROVED** [review]

**Comment by nickclark2016** [comment]
Thanks @starkos :)

**Pull request merged**
